src/components/MoviesRepo/MoviesRepo.js

```diff
@@ -1,43 +1,75 @@
 import React from "react"
 import Movie from "../Movie/Movie"
+import SelectedMovie from "../SelectedMovie/SelectedMovie"
 import './MoviesRepo.css'
 
 
-// class MoviesRepo extends React.Component {
-//     constructor(props) {
-//         super(props)
-//         this.state = {
-//             movies: props.movies
-//         }
-//     }
+class MoviesRepo extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            movies: props.movies,
+            selectedMovie: ""
+        }
+    }
 
-//     render() {
-//         const movieComponents = this.state.movies.map(movie => {
-//             return <Movie key={movie.id} image={movie.poster_path} title={movie.title}/>
-//         })
-//         return (
-//             <div className="movie-container">
-//                 {movieComponents}
-//             </div>
-//         )
-//     }
-// }
+    returnHome = () => {
+        this.setState({selectedMovie: ""})
+      }
+  
+    showMovieDetails = (id) => {
+        // console.log("details")
+        const foundMovie = this.state.movies.find(movie => {
+          return movie.id === id
+        })
+        this.setState({selectedMovie: foundMovie})
+    } 
 
-const MoviesRepo = props => {
-    const movieComponents = props.movies.map(movie => {
+    render() {
+        const movieComponents = this.state.movies.map(movie => {
             return <Movie 
                         key={movie.id} 
                         id={movie.id}
                         image={movie.poster_path} 
-                        showMovieDetails={props.showMovieDetails}
+                        showMovieDetails={this.showMovieDetails}
+                        returnHome={this.returnHome}
                     />
-    })
-    return (
-        <div className="movie-container">
-            {movieComponents}
-        </div>
+        })
+        return ( 
+            <>
+            {this.state.selectedMovie ? <SelectedMovie 
+                                            key={this.state.selectedMovie.id} 
+                                            poster={this.state.selectedMovie.poster_path} 
+                                            backdrop={this.state.selectedMovie.backdrop_path}
+                                            title={this.state.selectedMovie.title}
+                                            rating={this.state.selectedMovie.average_rating}
+                                            releaseDate={this.state.selectedMovie.release_date}
```

Comment:
Maybe consider declaring a variable to hold `this.state.selectedMovie` so you don't have to type that out so many times.